packages/opc-back-to-top/test/opc-back-to-top.test.ts

```diff
@@ -0,0 +1,39 @@
+import { LitElement } from 'lit-element';
+import { BackToTop } from '../src/opc-back-to-top';
+
+describe('opc-back-to-top', () => {
+
+    const OPC_COMPONENT = 'opc-back-to-top';
+    const ELEMENT_ID = 'opc-back-to-top';
+    let buttonElement: BackToTop;
+
+    const getShadowRoot = (tagName: string): ShadowRoot => {
```

Comment:
do  not use getElementsbyTagName please follow the test suite structure as defined in header component.
You need to use querySelector